stellarphot/settings/tests/test_models.py

```diff
@@ -284,6 +297,46 @@ def test_example(self, model, settings):
                     assert model_value == u.Unit(v)
                 else:
                     assert model_value == v
+        assert "fwhm_method" in TEST_PHOTOMETRY_OPTIONS
+
+
+class TestPriorVersionsCompatibility:
+    """
+    Make sure that all prior versions of settings files are still readable
+    by the current version of the code.
+
+    Each method in this test class should contain the release version number
+    of the version it is checking for compatibility.
+    """
+
+    @pytest.mark.parametrize(
+        "old_setting,new_setting",
+        (
+            [True, FwhmMethods.FIT],
+            [False, FwhmMethods.PROFILE],
+        ),
+    )
+    def test_2_0_0a11(self, old_setting, new_setting):
+        """
+        This version had fwhm_by_fit, which has now changed to fwhm_method.
+        """
+        assert "fwhm_method" in TEST_PHOTOMETRY_OPTIONS
```

Comment:
Does line 323 need to be repeated in line 325?

Reply:
Technically neither is needed...they are leftover from when I was trying to track down where `fwhm_method` was being changed. I'm going to merge this but will drop those lines in the next PR.